styles/lichess/catppuccin.user.css

```
@@ -0,0 +1,582 @@
+/* ==UserStyle==
+@name Lichess Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/lichess
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/lichess
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/lichess/catppuccin.user.css
+@description Soothing pastel theme for lichess
```

Oop I guess make Lichess capitalized here too
```suggestion
@description Soothing pastel theme for Lichess
```